config/settings.yml

```diff
@@ -1,4 +1,5 @@
 hold_button_url: https://myaccount.qa.k8s.libraries.psu.edu/holds/new?catkey=
+no_recall_button_url: https://myaccount.qa.k8s.libraries.psu.edu/no_recalls/new?catkey=
```

Comment:
I wonder if this could be refactored into `my_account_url` and have the code do something like `#{Settings.my_account_url}/holds/new?catkey=`

is no_recall `/ill/new?catkey=`?

Reply:
yeah that makes sense, I can make that change.

and yup no-recall is `/ill/new?catkey=?`